dlgr/griduniverse/experiment.py

```diff
@@ -1231,6 +1241,8 @@ def configure(self):
             (t["actor_start"], t["target_start"]): t
             for t in self.game_config.get("transitions", ())
         }
+
+        self.player_config = self.game_config.get("player_config")
```

Comment:
I think that at least for now, we'll need to JSON serialize `self.player_config` into `self.player_config_json` so the front-end can read it, but I'm not certain this is the the case (these may not actually be actually be used).

Reply:
We already pass the player color stuff directly, and the probability distribution is definitely server only. I think we're OK here for now, but if we eventually have client side player config in this structure, we'll probably need it. For now I don't think we do.